<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>CAN adapter settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Baudrate:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Encoding:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Filtering mode:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Filtering range</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 10px;
border-color: purple;
text-align: center;</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>5KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31K25BPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>125KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>250KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500KBPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000KBPS</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="encoding_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filtering_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="text">
     <string>All except</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>ID min:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>ID max:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accept_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>520</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_max">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_min">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Addres number base</string>
    </property>
   </widget>
   <widget class="QPushButton" name="base_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filtering_enable_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: white;
border-style: solid;
border-width: 5px;
border-radius: 25px;
border-color: purple;</string>
    </property>
    <property name="text">
     <string>Filtering off</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
